src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -189,13 +190,14 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
   const chainInfo = getChainInfo(token?.chainId)
   const networkLabel = chainInfo?.label
   const networkBadgebackgroundColor = chainInfo?.backgroundColor
+  const tokenDetailData: any = useTokenDetailQuery(address, 'ETHEREUM')
```

Comment:
Add a TODO comment so we know to come in later and update ETHEREUM to be from the selector